impl11/ddraw/DeviceResources.cpp

```diff
@@ -1502,7 +1504,7 @@ HRESULT DeviceResources::OnSizeChanged(HWND hWnd, DWORD dwWidth, DWORD dwHeight)
 			sd.BufferCount = 2;
 			sd.SwapEffect = DXGI_SWAP_EFFECT_SEQUENTIAL; // ORIGINAL = 0x1
 			//sd.SwapEffect = DXGI_SWAP_EFFECT_DISCARD;
-			//sd.SwapEffect = DXGI_SWAP_EFFECT_FLIP_SEQUENTIAL;
+			//sd.SwapEffect = DXGI_SWAP_EFFECT_FLIP_SEQUENTIAL; // Recommended for better performance. Can we use it?
```

Comment:
I _think_ I tried this, but didn't notice much difference in performance. Yes, I think we can try this for OpenXR

Reply:
I have enabled it, it does not seem to have any unwanted side effect, and it enables for example overlays (FPS)